Feature: dynamic commands

liveyourdream1 opened the pull request:
- Enables dynamic creation of commands from database entries (e.g., for starting custom commands)
- Added actions: show all commands, start all commands, refresh commands
- Implemented --start flag for commands

Commit:
Feature: dynamic commands

- Enables dynamic creation of commands from database entries (e.g., for starting custom commands)
- Added actions: show all commands, start all commands, refresh commands
- Implemented --start flag for commands

MaherRemita commented:
Hello, thanks for your contribution. Please contact me on Discord to discuss your pull request details because there is a lot to talk about 
My Discord: maher2003

Commit:
- fix tests
- add dynamic commands test
- remove --start flag
- commented-out example of dynamic_commands
